docs/content/installation/install_compatibility_requirements.md

```diff
@@ -51,7 +51,7 @@ Complete these steps to prepare your environment for installing the CSI (Contain
 
 2. Configure your storage system host attachment, per worker node.
 
-    **Important:** The CSI driver does not define hosts on your storage system.
+    **Note:** IBM® block storage CSI driver 1.10 introduces dynamic host definition. For more information and installation instructions, see [Installing the host definer](install_hostdefinition.md). If this feature is not installed, the CSI driver does not dynamically define hosts on your storage system and they must be defined manually.
```

Comment:
```suggestion
    **Note:** IBM® block storage CSI driver 1.10 introduces dynamic host definition. For more information and installation instructions, see [Installing the host definer](install_hostdefinition.md). If this feature is not installed, the nodes are not dynamically defined on your storage system and they must be defined manually.
```
since this feature is arguably not part of the CSI driver, I would avoid implying it, at least for now

Reply:
updated